apps/dot_ext/migrations/0002_authflowuuidcopy.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.24 on 2021-09-01 16:17
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
I'm a little unclear why this model is being added? Is this purely for testing purposes? I'm not seeing the part where it integrates back into the main user flow

Reply:
I checked No Migration on this PR, so this model is a copy of dot_ext_authflow_uuid with some slight modification e.g. allow "created" time stamp been assigned value..., this is used to help reviewers to try out and review, and this suppose to be removed after PR approved.